src/libaktualizr/package_manager/dockerappmanager.cc

```diff
@@ -3,12 +3,12 @@
 #include <sstream>
 
 struct DockerApp {
-  DockerApp(const std::string app_name, const PackageConfig &config)
-      : name(std::move(app_name)),
-        app_root(std::move(config.docker_apps_root / app_name)),
-        app_params(std::move(config.docker_app_params)),
-        app_bin(std::move(config.docker_app_bin)),
-        compose_bin(std::move(config.docker_compose_bin)) {}
+  DockerApp(std::string app_name, const PackageConfig &config)
+      : name(app_name),
```

Comment:
I'm also a bit surprised. It all looks fine except for this bit: why not move `app_name`? Seems like an unnecessary copy.

Reply:
Great question. I lost about 30 minutes of debugging time on that very issue. The problem is this line of code that comes *after* that std::move: `app_root(config.docker_apps_root / app_name),`. The app_name passed to that becomes an empty string and `app_root` winds up not being correct.

Reply:
Oh wow, I see. Can you not use `name` instead of `app_name` in the latter line?

Reply:
ah - better idea. i've done a force-push with rebase on master + plus this suggestion.